README.md

```diff
@@ -1 +1,84 @@
-# first-capstone-project-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+​
+# Capstone Project 1
+​
+> This project was designed to test some of the abilities I learned in the first module of the Microverse program. I had to create a website from scratch following basic [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). The abilities to be tested are: Use semantic HTML tags.
+- Apply best practices in HTML code.
+- Use CSS selectors correctly.
+- Use CSS box model.
+- Use Flexbox to place elements in the page.
+- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
+- Use GitHub Pages to deploy web pages.
+- Apply JavaScript best practices and language style guides in code.
+- Use JavaScript to manipulate DOM elements.
+- Use JavaScript events.
+- Use objects to store and access data.
+- Communicate technical concepts to other technical people.
+​
+The requirements for this projects were:
+* Build only these 2 pages: The "home" page and the "about" page.
+​
+* Each of these pages should have versions for 2 different screen sizes: mobile: up to 768px wide and desktop: 768px or wider.
+​
+* Interactions
+    - Links:
+      - The home page should have a link in the menu to the about page.
+      - The logo in the header links to the home page.
+      
+    - Mobile menu
+      - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
+      - There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
+      - The mobile menu should have a close (X) button that closes the menu.
+​
+*Dynamic page
+  - The section "Featured speakers" should be created dynamically in JavaScript.
+  - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.
+​
+## Built With
+​
+- HTML
+- CSS
+- JavaScript
+​
+​
+## Live Demo
+​
+[Live Demo Link]()
```

Comment:
- [ ] Kindly make sure to deploy your capstone with Github Pages and provide the link here.
- [ ] The project is deployed properly, all pages, links, and images work as expected.